app/controllers/account_invitations_controller.rb

```diff
@@ -2,16 +2,16 @@ class AccountInvitationsController < ApplicationController
   before_action :authenticate_user!
```

Comment:
Just leave this controller in the code, it should be alone, but nest it inside to `accounts/:id` route.